diff --git a/dist/index.mjs b/dist/index.mjs
index 1f2dbeb383466f53c37134f3f7072c6f4609bed5..19e739594606014c0ce6308200bd517c315c3586 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -53005,6 +53005,7 @@ const outputConfigPlugin = createPlugin("output-config", (ctx) => {
 				const isEntryWorker = ctx.resolvedPluginConfig.type === "workers" && this.environment.name === ctx.resolvedPluginConfig.entryWorkerEnvironmentName;
 				outputConfig = {
 					...inputWorkerConfig,
+					pages_build_output_dir: void 0,
 					main: entryChunk.fileName,
 					no_bundle: true,
 					rules: [{
